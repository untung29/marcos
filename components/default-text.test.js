import React from "react";
import renderer from "react-test-renderer";
// import { shallow } from "enzyme";
import DefaultText from "./default-text";

it("<DefaultText />", () => {
  const tree = renderer.create(<DefaultText />).toJSON();
  expect(tree).toMatchSnapshot();
});
