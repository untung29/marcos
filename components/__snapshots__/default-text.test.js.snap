// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DefaultText /> 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "white",
        "fontFamily": "PoppinsRegular",
      },
      undefined,
    ]
  }
/>
`;
